libparsec/crates/crypto/src/sodiumoxide/mod.rs

```diff
@@ -17,3 +17,7 @@ pub use secret::*;
 pub use sequester::*;
 pub use sign::*;
 pub(crate) use utils::*;
+
+pub fn init() -> anyhow::Result<()> {
+    sodiumoxide::init().map_err(|_| anyhow::anyhow!("Failed to init sodiumoxide"))
```

Comment:
nitpick: I would rather use `map_err(|()| anyhow::...`, the `()` instead of `_` makes it obvious that `sodiumoxide::init()` doesn't return a precise error (since in that case it that would be convenient to add it to the returned error)

Reply:
Also, since we just panic in `libparsec_init()` instead of returning the error through the bindings, we should just move the panic here (given it simplifies the code, and sodiumoxide is never supposed to not be able to initialize)

Reply:
done